samples/csharp_dotnetcore/12.customQABot/CognitiveModels/SampleForCQA.tsv

```diff
@@ -0,0 +1,157 @@
+Question	Answer	Source	Metadata	SuggestedQuestions	IsContextOnly	Prompts	QnaId	SourceDisplayName
```

Comment:
Are we using the same file in all four samples (JS and dotnet), if yes, can we use the same  file name for all of them?

Reply:
We will be using same file for both samples in JS and dotnet and file name would remain the same ("SampleForCQA.tsv").